kinetic/distribution.yaml

```diff
@@ -3880,7 +3880,11 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/UbiquityRobotics-release/magni_robot-release.git
-      version: 0.1.1-0
+      version: 0.2.0-0
+    source:
+      type: git
+      url: git
+      version: https://github.com/UbiquityRobotics/magni_robot.git
```

Comment:
I did use bloom, but looks like I screwed up entering the info or something, sorry about that.